core/src/main/java/org/infinispan/interceptors/xsite/NonTransactionalBackupInterceptor.java

```diff
@@ -90,6 +96,9 @@ private WriteCommand transform(DataWriteCommand command) {
       } else if (command instanceof ComputeIfAbsentCommand) {
          ComputeIfAbsentCommand computeIfAbsentCommand = (ComputeIfAbsentCommand) command;
          return commandsFactory.buildComputeIfAbsentCommand(computeIfAbsentCommand.getKey(), computeIfAbsentCommand.getMappingFunction(), computeIfAbsentCommand.getMetadata(), computeIfAbsentCommand.getFlagsBitSet());
+      } else if (command instanceof ReadWriteKeyCommand) {
+         ReadWriteKeyCommand readWriteKeyCommand = (ReadWriteKeyCommand) command;
```

Comment:
You can't backup functional commands (compute handling is wrong, too), but I'll handle that in ISPN-7961.